assets/terraform-modules/bootkube/conditional.tf

```diff
@@ -26,3 +26,14 @@ resource "local_file" "calico" {
   })
   filename = "${var.asset_dir}/charts/kube-system/calico.yaml"
 }
+
+# Populate packet-ccm chart values file named packet-ccm.yaml.
```

Comment:
This should be added the same way as `calico-host-protection`, not directly in `bootkube` module.